### DISCUSSION TITLE
Installation Questions

### DISCUSSION THREAD

**Discussion by sgoggins** [chaoss/augur#1233] [Q&A]
Ask questions about installing Augur.

**Comment by shamim27khan** [comment]
i am trying to use docker-setup.sh,

The script never exits, and here are the trimmed logs 

backend_1  | [78] augur.routes.broker [INFO] Worker workers.pull_request_worker.50449 is idle, preparing to send the pull_request_files model for url: https://github.com/SociallyCompute/update-test task to http://0.0.0.0:50449/AUGWOP/task
backend_1  |
backend_1  | [91] augur.routes.broker [INFO] Message received that worker workers.pull_request_worker.50449 completed task: {'worker_id': 'workers.pull_request_worker.50449', 'job_type': 'MAINTAIN', 'repo_id': 25430, 'job_model': 'pull_request_files', 'github_url': 'https://github.com/SociallyCompute/update-test'}
backend_1  |
backend_1  | [91] augur.routes.broker [INFO] Worker workers.pull_request_worker.50449 is idle, preparing to send the pull_request_commits model for url: https://github.com/chaoss/grimoirelab-perceval-opnfv task to http://0.0.0.0:50449/AUGWOP/task
backend_1  |
backend_1  | [119] augur.routes.broker [INFO] Message received that worker workers.pull_request_worker.50449 completed task: {'worker_id': 'workers.pull_request_worker.50449', 'job_type': 'MAINTAIN', 'repo_id': 25445, 'job_model': 'pull_request_commits', 'github_url': 'https://github.com/chaoss/grimoirelab-perceval-opnfv'}
backend_1  |
backend_1  | [119] augur.routes.broker [INFO] Worker workers.pull_request_worker.50449 is idle, preparing to send the pull_requests model for url: https://github.com/chaoss/augur task to http://0.0.0.0:50449/AUGWOP/task
backend_1  |
backend_1  | [95] augur.routes.broker [INFO] Message received that worker workers.github_worker.57856 completed task: {'worker_id': 'workers.github_worker.57856', 'job_type': 'MAINTAIN', 'repo_id': 24442, 'job_model': 'issues', 'github_url': 'https://github.com/operate-first/blueprint'}
backend_1  |
backend_1  | [95] augur.routes.broker [INFO] Worker workers.github_worker.57856 is idle, preparing to send the issues model for url: https://github.com/SociallyCompute/update-test task to http://0.0.0.0:57856/AUGWOP/task

What next should i do?
